python_function/task_4_arg_valid_func.py

```diff
@@ -0,0 +1,51 @@
+"""Функція з іменованими аргументами та валідацією
+Функцію create_user(username, email, password, **kwargs), яка:
+Приймає username, email та password як обов'язкові параметри
+Приймає будь-які додаткові дані користувача через **kwargs (наприклад, вік, адреса)
+
+Якщо валідація успішна, повертає словник з даними користувача,  
+або повідомлення про помилку, якщо валідація не вдалася"""
+
+"""Запит для вводу данних користувачем"""
+username = input("Введить ім'я не менше 5 символів: ")
+email = input("Введить email: ")
+password = input("Введить пароль не менше 8 символів: ")
+
+errors = []  # Список для збереження помилок
```

Comment:
use `set()` to avoid duplicate errors
```suggestion
errors = set()
```